client/login/magic-login/emailed-login-link-successfully.jsx

```diff
@@ -63,24 +57,34 @@ class EmailedLoginLinkSuccessfully extends Component {
 					waitForEmailAddress={ emailAddress }
 				/>
 
-				<h1 className="magic-login__form-header">{ translate( 'Check your email!' ) }</h1>
+				<h1 className="magic-login__form-header">{ translate( 'Check your email' ) }</h1>
 
 				<Card className="magic-login__form">
-					<p>{ preventWidows( line ) }</p>
+					<div className="magic-login__form-text">
+						<p>{ preventWidows( line ) }</p>
+						<b>{ emailAddress }</b>
+					</div>
 				</Card>
+				<div className="magic-login__emails-list">
+					<MagicLoginEmailWrapper emailAddress={ emailAddress } />
+				</div>
 
 				<div className="magic-login__footer">
-					<a
-						href={ login( {
-							isJetpack: this.props.isJetpackLogin,
-							isWhiteLogin: this.props.isWhiteLogin,
-							locale: this.props.locale,
-						} ) }
-						onClick={ this.onClickBackLink }
-					>
-						<Gridicon icon="arrow-left" size={ 18 } />
-						{ translate( 'Back to login' ) }
-					</a>
+					<p>
+						{ translate(
+							" Didn't get the email? You might want to double check if the email address is associated with your account,"
+						) }
+						<a
+							href={ login( {
+								isJetpack: this.props.isJetpackLogin,
+								isWhiteLogin: this.props.isWhiteLogin,
+								locale: this.props.locale,
+							} ) }
+							onClick={ this.onClickBackLink }
+						>
+							{ translate( 'or login with a password instead.' ) }
```

Comment:
The original version is probably better. Some accounts are passwordless.

Reply:
Would you mind elaborating a bit more on what you see as improvements?

Reply:
Not all accounts have passwords, "use password instead" isn't accurate.

Reply:
Maybe instead of `or login with a password instead`. And point back to log in, we could say something like or try to reset your password and point the user to https://wordpress.com/wp-login.php?action=lostpassword
WDYT?

Reply:
@crisbusquets I would be truly interested in your opinion on this one too :)

Reply:
Oh, I see. Instead of assuming they've lost their password (maybe they don't even have a password, as @alshakero mentioned), I'd use the magic link (https://wordpress.com/log-in/link).

What about:

`Didn't get the email? You might want to double check if the email address is associated with your account, or [request a login link] to log you in right away.`

PS: Wow, this interface (https://wordpress.com/wp-login.php?action=lostpassword) is super old!

Reply:
@crisbusquets thank you for your input. I think in the context of the message and where users are, requesting a magic link. We should either help users if they did not get an email by sending them to [lostpassword] (https://wordpress.com/wp-login.php?action=lostpassword). Yes, it looks terrible, but it is in the roadmap https://github.com/Automattic/wp-calypso/issues/33924 or/and just point them back to Log-in, which does not help much but make sense IMO.

If I understood correctly, The message you proposed in this context can create an infinite loop since they are already requesting a login link, aka, magic link.

WDYT?

Reply:
> @crisbusquets thank you for your input. I think in the context of the message and where users are, requesting a magic link. We should either help users if they did not get an email by sending them to [lostpassword] (https://wordpress.com/wp-login.php?action=lostpassword). Yes, it looks terrible, but it is in the roadmap #33924 or/and just point them back to Log-in, which does not help much but make sense IMO.
> 
> If I understood correctly, The message you proposed in this context can create an infinite loop since they are already requesting a login link, aka, magic link.
> 
> WDYT?

Ahh... yes. For some reason, I didn't see that we were already on the magic link flow, sorry. Too many sign-up and login flows hahahaa

We need to be able to provide a way to log in even if the magic link fails. The lost password flow (what you initially suggested) could work, and it seems the only solution we can offer 🤔 because asking them to use the Jetpack app will overcomplicate everything. Just to know, do we know how many of our users have no password?

Reply:
Can we do `Didn't get the email? You might want to double check if the email address is associated with your account, or reset your password` ? Pointing to [this](https://wordpress.com/wp-login.php?action=lostpassword) 
Regarding how many users have no password, I will check that info and get back to you soon.

Reply:
@crisbusquets @renancarvalho  I think for this convo it’s enough to understand we have a lot of both kinds of users, knowing more won’t really inform the way or the other? :-) 

If we have their email, we know if they're passwordless, and should be able to adjust copy accordingly? They won't be able to create a new password without access to email anyway, and they should contact support.